### PR TITLE
Use standard indexes.

### PR DESCRIPTION
**Pull request by tomprince**
This depends on https://phabricator.services.mozilla.com/D64152 landing for PRs to work.

### CODE REVIEW

**Comment by La0** [comment]
Are those standard index format documented somewhere ? I thought the `project.X.Y.Z` was still valid as [it's still used by many tasks](https://firefox-ci-tc.services.mozilla.com/tasks/index/project)

**Comment by tomprince** [comment]
> Are those standard index format documented somewhere ? I thought the `project.X.Y.Z` was still valid as [it's still used by many tasks](https://firefox-ci-tc.services.mozilla.com/tasks/index/project)

It is still used in some places, but we grant `{trust_domain}.v2.{alias}.*` to repos via the `trust-domain-scopes` features, (along with generic scopes for worker-types and caches). What goes after that prefix isn't well defined[1], and I just picked something similar to what you had under `project.*`. I don't have strong feelings about anything other than using the new prefix that doesn't require additional grants.

[1] taskgraph has some defaults, but doesn't enforce them. https://hg.mozilla.org/ci/taskgraph/file/tip/src/taskgraph/transforms/task.py#l189 are the default indexes that taskgraph creates (but note they are based on hg rather than github projects)

File: .taskcluster.yml [hunk 1]
```diff
@@ -146,10 +149,14 @@ tasks:
...
-                then: "index.project.code-review.bot.pull_request.${head_branch}"
-                else: "index.project.code-review.bot.${head_branch}"
+              $if: 'tasks_for == "github-pull-request"'
+              then:
+                - "index.code-analysis.v2.code-review-pr.revision.${head_rev}"
+                - "index.code-analysis.v2.code-review-pr.branch.${head_branch}"
+              else:
+                - "index.code-analysis.v2.code-review.revision.${head_rev}"
+                - "index.code-analysis.v2.code-review.branch.${head_branch}"
+                - "code-analysis.v2.code-review.revsion.${head_rev}"
```
Comment (La0) [inline .taskcluster.yml]:
```suggestion

```

File: .taskcluster.yml [hunk 2]
```diff
@@ -146,10 +149,14 @@ tasks:
...
             scopes:
               - queue:route:index.project.code-review.bot.*
```
Comment (La0) [inline .taskcluster.yml]: This scope needs to be removed or updated

**Comment by La0** [comment]
I can't update the pr on your fork, i may re-create it in order to fix the scope request

**Pull request merged**